File: GLCalendarAndroid/app/src/main/java/com/example/glcalendarandroid/GoogleUserAppointment.java
package com.example.glcalendarandroid;

import com.google.api.client.util.DateTime;

import java.util.Date;

public class GoogleUserAppointment {
    public String Id;
    public String Summary;
    public Date StartTime;
    public Date EndTime;

    public GoogleUserAppointment(String Id, String Summary, long StartTime, long EndTime) {
        this.Id = Id;
        this.Summary = Summary;
        this.StartTime = new Date(StartTime);
        this.EndTime = new Date(EndTime);
    }

}

File: GoogleCalendar (xamarin + native)/GoogleCalendar/GoogleCalendar/GoogleUserAppointment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GoogleCalendar {
    public class GoogleUserAppointment {
        public string Id { get; set; }
        public string Summary { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
    }
}
